core/platform/search/elastic_search_services.py

```diff
@@ -0,0 +1,159 @@
+# coding: utf-8
+#
+# Copyright 2020 The Oppia Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the 'License');
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an 'AS-IS' BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Provides platform search services implemented using the elastic search python
+API.
+"""
+
+from __future__ import absolute_import  # pylint: disable=import-only-modules
+from __future__ import unicode_literals  # pylint: disable=import-only-modules
+
+import json
+
+import elasticsearch
```

Comment:
elasticsearch should be declared as a third-party lib in isort.cfg, and put in a separate import section because it does not originate from oppia.

Reply:
Done.

Reply:
This wasn't done properly. I've fixed it now.